helm/reana/templates/secrets.yaml

```diff
@@ -23,6 +23,19 @@ data:
   CERN_CONSUMER_KEY: {{ .Values.secrets.cern.sso.CERN_CONSUMER_KEY | default "cern_consumer_key" | b64enc }}
   CERN_CONSUMER_SECRET: {{ .Values.secrets.cern.sso.CERN_CONSUMER_SECRET | default "cern_consumer_secret" | b64enc }}
 ---
+{{- if .Values.login }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "reana.prefix" . }}-login-provider-secrets
```

Comment:
extremely minor: let's call this `...-login-providers-secrets` for consistency with `LOGIN_PROVIDERS_SECRETS`?